lib/ngx/ssl.lua

```diff
@@ -164,6 +167,26 @@ function _M.server_name()
     return nil, ffi_str(errmsg[0])
 end
 
```

Comment:
Style: we use 2 blank lines to separate Lua function definitions consistently in this Lua library.